modules/020-cni-cilium/templates/namespace.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: d8-{{ .Chart.Name }}
```

Comment:
You need to pick one variant and stick to it: `d8-{{ .Chart.Name }}` or `d8-cni-cilium`.

Reply:
Done.